cucumber-jparest/src/main/java/uk/gov/homeoffice/digital/sas/cucumberjparest/JpaTestContext.java

```diff
@@ -0,0 +1,113 @@
+package uk.gov.homeoffice.digital.sas.cucumberjparest;
+
+import static java.util.Map.entry;
+import static uk.gov.homeoffice.digital.sas.cucumberjparest.ServiceRegistry.SERVICE_REGISTRY_SYSTEM_PROPERTY_NAME;
+
+import io.cucumber.spring.ScenarioScope;
+import java.time.Instant;
+import java.util.List;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.homeoffice.digital.sas.cucumberjparest.utils.SerialisationUtil;
+
+/**
+ * Class used by the {Link ContextConfiguration} annotation to configure an
+ * {@link org.springframework.context.ApplicationContext ApplicationContext} for integration tests.
+ * In this case for the cucumber tests.
+ */
+@Configuration
+public class JpaTestContext {
+
+  public static final Map<String, Class<?>> classSimpleStrings = Map.ofEntries(
+      entry("String", String.class),
```

Comment:
nit: can use `String.class.getSimpleName()` instead of the hardcoded one, same for the other entries.
Also could probs get away with just `Map.of(..)` without the entries.
Suggestion is completely optional and doesn't provide much benefit, but thought it was worth mentioning.

Reply:
Good point. Actioned!